aws-storage-s3/api/aws-storage-s3.api

```diff
@@ -329,8 +331,8 @@ public abstract interface class com/amplifyframework/storage/s3/service/StorageS
 	public abstract fun downloadToFile (Ljava/lang/String;Ljava/lang/String;Ljava/io/File;Z)Lcom/amplifyframework/storage/s3/transfer/TransferObserver;
 	public abstract fun getPresignedUrl (Ljava/lang/String;IZ)Ljava/net/URL;
 	public abstract fun getTransfer (Ljava/lang/String;)Lcom/amplifyframework/storage/s3/transfer/TransferRecord;
-	public abstract fun listFiles (Ljava/lang/String;Ljava/lang/String;)Ljava/util/List;
-	public abstract fun listFiles (Ljava/lang/String;Ljava/lang/String;ILjava/lang/String;)Lcom/amplifyframework/storage/result/StorageListResult;
+	public abstract fun listFiles (Ljava/lang/String;Ljava/lang/String;ILjava/lang/String;Lcom/amplifyframework/storage/options/SubpathStrategy;)Lcom/amplifyframework/storage/result/StorageListResult;
```

Comment:
We can see that we're changing the API here - the changes are not additive. There should only ever be additions to `.api` files - modifications or deletions indicate a breaking change. See other comments for how to address.

Reply:
Should be updated now to maintain backward compatibility and not introduce breaking changes.